packages/eslint-config-mc-app/index.js

```diff
@@ -73,6 +73,7 @@ module.exports = {
     'no-underscore-dangle': 0,
     'jest/no-identical-title': 'warn',
     'jest/no-focused-tests': 2,
+    /* eslint-plugin-react */
```

Comment:
Is this comment intended?

Reply:
Yes, my intent is to group the rules of the react plugin using that comment, so it's easier to spot them.